hs_core/tests/api/native/test_core_metadata.py

```diff
@@ -1062,20 +1062,12 @@ def test_relation(self):
         self.assertRaises(Exception, lambda: resource.update_metadata_element(self.res.short_id, 'relation',
                                                                               rel_to_update.id, type='isCopiedFrom'))
 
-        # test that duplicate relation types are not allowed
-        self.assertRaises(Exception, lambda: resource.create_metadata_element(
```

Comment:
@pkdash 
we allow user to add multiple relations of the same type.  I think we dont care if the values are same as well. although it seems useless, it is user's freedom to add duplicated values, just like he can create multiple identical resources.